File: code_snippets/ts/src/develop/kafka.ts
import * as restate from "@restatedev/restate-sdk";

const service = restate.object({
  name: "KafkaService",
  handlers: {
    myEventHandler: async (ctx: restate.Context, name: string) =>
      // <start_headers>
      ctx.request().headers,
    // <end_headers>
  },
});

File: code_snippets/go/develop/kafka.go
package develop

import restate "github.com/restatedev/sdk-go"

type KafkaService struct{}

func (KafkaService) MyEventHandler(ctx restate.Context, name string) error {
	_ =
		// <start_headers>
		ctx.Request().Headers
	// <end_headers>

	return nil
}
